packages/component-library/components/Heading/Heading.tsx

```diff
@@ -59,8 +60,12 @@ export const Heading = ({
   tag,
   variant,
   color = "dark",
+  automationId,
   ...otherProps
 }: HeadingProps) => {
+  const dataAutomationId = automationId
+    ? { "data-automation-id": automationId }
+    : { "data-automation-id": "" }
```

Comment:
@Confidenceman02 I think you could set this to `undefined` rather than an empty string in the case that the `automationId` isn’t passed — should mean that the attribute doesn’t render at all, rather than rendering with no value.

Reply:
looks like we don't even need this! data-automation-id can be passed without it existing on the props.

Reply:
Ah yeah, of course.